src/block-components/progress-bar/style.js

```diff
@@ -1,43 +1,49 @@
-import { useStyles, getStyles } from '~stackable/util'
+import {
+	useStyles, getStyles, hexToRgba,
+} from '~stackable/util'
 import { Style as StyleComponent } from '~stackable/components'
 import {
 	DEFAULT_PERCENT, DEFAULT_SIZE, DEFAULT_THICKNESS,
 } from './attributes'
 
-const getStyleParams = () => {
+const getStyleParams = ( { isCircle } ) => {
 	return [
 		{
-			selector: '.stk-progress-circle',
-			styleRule: '--progress-thickness',
-			attrName: 'progressThickness',
-			format: '%spx',
+			selector: '[class*="stk-progress-"]',
```

Comment:
**Attribute contains** selectors are relatively slow. I'd like to suggest that instead of using `[class*="stk-progress-"]`, just change the selectors directly:

```js
const selector = isCircle ? '.stk-progress-circle' : '.stk-progress-bar'
```

```diff
- selector: '[class*="stk-progress-"]',
+ selector,
```

```